syne_tune/remote/remote_launcher.py

```diff
@@ -185,8 +185,8 @@ def remote_script_dir(self) -> Path:
     def launch_tuning_job_on_sagemaker(self, wait: bool):
         # todo add Sagemaker cloudwatch metrics to visualize live results of tuning best results found over time.
         if self.instance_type != "local":
-            checkpoint_s3_root = f"{self.s3_path}/"
-            logger.info(f"Tuner will checkpoint results to {checkpoint_s3_root}{self.tuner.name}")
+            checkpoint_s3_root = f"{self.s3_path}/{self.tuner.name}"
+            logger.info(f"Tuner will checkpoint results to {checkpoint_s3_root}")
```

Comment:
Thanks, exactly what I thought is the best solution

Reply:
Awesome thanks for your patience for this fix.